README.md

```diff
@@ -74,6 +74,9 @@ more information.
 
         $ make run
 
+5. When signing up for the app locally, you can view sent system emails at
+   http://localhost:1080/.
+
```

Comment:
Ok, updated the organization. Let me know what you think! I imagine eventually we will want to move all of this to its own markdown folder.